drivers/flash/Kconfig.cadence_qspi_nor

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2022 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_CAD_QSPI_NOR := cdns,qspi-nor
+
+config FLASH_CAD_QSPI_NOR
+	bool "Cadence Quad SPI Flash driver"
+	depends on FLASH
```

Comment:
remove `depends on FLASH` is implied since this file wouldn't get sourced otherwise.

Reply:
Resolved, Thanks.